component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -689,32 +750,37 @@ private void deployServers() throws ConnectionUnavailableException {
                 properties.put(Constants.MODE, mode);
                 properties.put(Constants.HEADER_PRESENT, headerPresent);
                 properties.put(Constants.READ_ONLY_HEADER, readOnlyHeader);
+                properties.put(Constants.CRON_EXPRESSION, cronExpression);
```

Comment:
We arent using the 'put' Constants.CRON_EXPRESSION right? Shall we remove this if so?

Reply:
fixed in 47cc199b8188f3701d228a031aeb7a33f02bc856